packages/build-utils/package.json

```diff
@@ -11,7 +11,11 @@
     "directory": "packages/now-build-utils"
   },
   "scripts": {
-    "build": "node build",
+    "build": "run-p build:js build:types",
+    "build:js": "run-s build:single build:bundle",
+    "build:single": "node ../../scripts/esbuild.mjs",
+    "build:bundle": "node ../../scripts/esbuild.mjs --bundle",
```

Comment:
Since these both have the same output, is it going to cause conflicts depending on which finishes first?

Reply:
No, because `build:js` uses `run-s` (serial invocation), so the single-file output happens first (`tsc` equivalent), and then the bundle output happens second (`ncc` equivalent).

Reply:
Did you do a diff on the `dist` directory before and after to see if anything changed? For example, do we need to move any asset files?

Reply:
I did, and there's no external files used in build-utils. The file counts are identical:

```
$ ls dist.old
clone-env.d.ts      errors.js         fs/                         hard-link-dir.d.ts  nodejs-lambda.js   types.d.ts
clone-env.js        file-blob.d.ts    get-ignore-filter.d.ts      hard-link-dir.js    prerender.d.ts     types.js
debug.d.ts          file-blob.js      get-ignore-filter.js        index.d.ts          prerender.js       validate-npmrc.d.ts
debug.js            file-fs-ref.d.ts  get-platform-env.d.ts       index.js            schemas.d.ts       validate-npmrc.js
edge-function.d.ts  file-fs-ref.js    get-platform-env.js         lambda.d.ts         schemas.js
edge-function.js    file-ref.d.ts     get-prefixed-env-vars.d.ts  lambda.js           should-serve.d.ts
errors.d.ts         file-ref.js       get-prefixed-env-vars.js    nodejs-lambda.d.ts  should-serve.js

$ ls dist.new
clone-env.d.ts      errors.js         fs/                         hard-link-dir.d.ts  nodejs-lambda.js   types.d.ts
clone-env.js        file-blob.d.ts    get-ignore-filter.d.ts      hard-link-dir.js    prerender.d.ts     types.js
debug.d.ts          file-blob.js      get-ignore-filter.js        index.d.ts          prerender.js       validate-npmrc.d.ts
debug.js            file-fs-ref.d.ts  get-platform-env.d.ts       index.js            schemas.d.ts       validate-npmrc.js
edge-function.d.ts  file-fs-ref.js    get-platform-env.js         lambda.d.ts         schemas.js
edge-function.js    file-ref.d.ts     get-prefixed-env-vars.d.ts  lambda.js           should-serve.d.ts
errors.d.ts         file-ref.js       get-prefixed-env-vars.js    nodejs-lambda.d.ts  should-serve.js

$ ls dist.old/fs
download.d.ts                get-writable-directory.js  node-version.d.ts    normalize-path.js      rename.d.ts            run-user-scripts.js
download.js                  glob.d.ts                  node-version.js      read-config-file.d.ts  rename.js              stream-to-buffer.d.ts
get-writable-directory.d.ts  glob.js                    normalize-path.d.ts  read-config-file.js    run-user-scripts.d.ts  stream-to-buffer.js

$ ls dist.new/fs
download.d.ts                get-writable-directory.js  node-version.d.ts    normalize-path.js      rename.d.ts            run-user-scripts.js
download.js                  glob.d.ts                  node-version.js      read-config-file.d.ts  rename.js              stream-to-buffer.d.ts
get-writable-directory.d.ts  glob.js                    normalize-path.d.ts  read-config-file.js    run-user-scripts.d.ts  stream-to-buffer.js
```